src/components/CheckForm/AlertCard.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { Controller, useFormContext } from 'react-hook-form';
+import { Card, Field, Icon, Input } from '@grafana/ui';
+
+import { CheckAlertType, CheckFormValues } from 'types';
+
+import { useCheckFormContext } from './CheckFormContext/CheckFormContext';
+
+interface AlertCardProps {
+  predefinedAlert: { type: CheckAlertType; description: string };
+  onSelect: (type: CheckAlertType, forceSelection?: boolean) => void;
+}
+
+export const AlertCard = ({ predefinedAlert, onSelect }: AlertCardProps) => {
+  const { control, formState, getValues } = useFormContext<CheckFormValues>();
+  const { isFormDisabled } = useCheckFormContext();
+
+  const thresholdError = formState.errors?.alerts?.[predefinedAlert.type]?.threshold?.message;
+
+  const isSelected: boolean = getValues(`alerts.${predefinedAlert.type}.isSelected`) || false;
+  const threshold: number = getValues(`alerts.${predefinedAlert.type}.threshold`) || 0;
+
+  return (
+    <Card
```

Comment:
I'll do a much more thorough review of the UI/UX separately but just a heads up I don't think we should use cards but probably a table of some sort.